TundraTD/Assets/Scripts/SceneManagement/LevelsScene.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+namespace SceneManagement
+{
+    public class LevelsScene : MonoBehaviour
+    {
+        [SerializeField] private RectTransform buttonsHolder;
+        [SerializeField] private Button levelButtonPrefab;
+        private int _levelsAmount;
+        private int _currentLevelBindingIndex = 1;
+        private const int LevelsBindingStartIndex = 2;
+        
+        private void Start()
+        {
+            if (_levelsAmount != 0)
+                return;
+            
+            CountLevelsInBuild();
+            print(_levelsAmount);
+
+            InstantiateLevelButtons();
+        }
+
+        private void InstantiateLevelButtons()
+        {
+            for (int i = 0; i < _levelsAmount; i++)
+            {
+                var button = Instantiate(levelButtonPrefab, buttonsHolder.transform);
+                button.GetComponentInChildren<Text>().text = $"Level{i + 1}";
```

Comment:
What about printing the space between level name and number?

Reply:
It's just a filler for a beautiful sprite